src/search_procedure.jl

```diff
@@ -12,7 +12,7 @@ function search(g::Grammar, problem::Problem, depth::Int, start::Symbol, enumera
 
         # Evaluate examples the examples.
         #  `evaluate examples` returns as soon as it has found the first example that doesn't work.
-        if Evaluation.evaluate_examples(symboltable, expr, problem.examples)
+        if test_examples(symboltable, expr, problem.examples)
```

Comment:
Maybe it would be nice to update the comment with `test_example` or delete if it is no longer necessary.